docs/FreeSWITCH-Explained/Troubleshooting-Debugging/Energy-Levels-Silence-Thresholds-Silence-Hits.mdx

```diff
@@ -10,39 +10,36 @@ tags:
 
 # Energy Levels, Silence Thresholds, Silence Hits 
 
- 
-
 ## About
 
-This article is aimed at guiding you through source code modifications to you help you discover the actual reported values FreeSWITCH uses to determine silence and modify your threshold/hits values accordingly to provide optimal usage of silence detection. This particular exercise will modify the "record" app as provided by mod\_dptools.
-
+This article is aimed at guiding you through source code modifications to you help you discover the actual reported values FreeSWITCHⓇ uses to determine silence and modify your threshold/hits values accordingly to provide optimal usage of silence detection. This particular exercise will modify the "record" app as provided by `mod_dptools`.
 
 ## Locate your specific line in source, where we'll add more lines
 
-There are several areas in source code that deal with engergy levels, silence threshold, and silence hits (Like in wait\_for\_silence, or conference) but were specifically gonna target "record" area of source code for this modification.
+There are several areas in source code that deal with engergy levels, silence threshold, and silence hits (like in `wait_for_silence`, or conference) but we're specifically gonna target "record" area of source code for this modification.
 
-```xml
+```bash title="terminal session"
```

Comment:
Added titles in a lot of place and also tried to set the correct lang for the fences.